vars/mavenNode.groovy

```diff
@@ -5,6 +5,18 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def nodesDir = "/var/lib/jenkins/nodes"
+    def file = new File(nodesDir)
+    
+    if(file.isDirectory()){
+        echo "$nodesDir exists and it's a directory"
+    } else {
+        error """
+        Directory $nodesDir does not exist. /nodes directory stores data for managing slave nodes.
```

Comment:
does it means `/nodes directory ` -> `$nodesDir directory ` in second lin?

If yes then kee it consistent.

Reply:
`nodes` here is the logical name of a directory. If I put path instead and say that it stores data, it would be wrong, as the directory doesn't exist in this case.